src/apple/xcode-manager.ts

```diff
@@ -255,18 +264,38 @@ function addUploadSymbolsScript(
 }
 
 export class XcodeProject {
```

Comment:
`m`: This class is huge. Can we maybe break it up into smaller chunks?

Reply:
I agree it is massive, I might be able to split it up into reasonable parser/reader utility classes, but in general it is just a wrapper around the `XcodeProject` type

Reply:
I leave this up to you. No strong opinion.

Reply:
I'll refactor it in another PR